druzhba/mssql.py

```diff
@@ -50,10 +50,14 @@ def connection_vars(self):
         }
 
     def get_sql_description(self, sql):
+        table_attributes = {}  # TODO: retrieve table attributes
         with closing(pymssql.connect(**self.connection_vars)) as conn:
             with closing(conn.cursor(as_dict=True)) as cursor:
                 cursor.execute(sql)
-                return ((col[0], col[1]) for col in cursor.description)
+                return (
```

Comment:
it would unfortunately take a larger refactor to replace this return type with a shared data structure across all the database types